scripts/calculate_rating.py

```diff
@@ -0,0 +1,151 @@
+'''
+Implementation of following rating algorithm
+https://codeforces.com/blog/entry/20762
+'''
+
+import datetime
+import pymysql
+import json
+from pymysql.cursors import DictCursor
+
+import itertools
+import pprint
+
+USER_QUERY = 'SELECT user_id, username from users'
+
+# TODO technically if a user has the same start time, they'll show up twice, this could be an issue
```

Comment:
Like if a user starts two runs at the same time? Not sure about the scenario here. If it's a matter of uniqueness, perhaps another GROUP BY after the final ORDER BY might work?

Reply:
Yeah if they do the exact time, or somehow submit a duplicate, I think maybe let's put this in as an issue and figure something out later